cityofaustin/atd-moped
# Create projects map page layout and route

## This is the collector branch for the projects map MVP

This will not be merged until the MVP is complete and ready for release.

### Merge TODOs
- [x] Add to QA test script
- [x] Merge main and test migrations (update if needed)

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9913
https://github.com/cityofaustin/atd-data-tech/issues/9914
https://github.com/cityofaustin/atd-data-tech/issues/9919

This adds: 
- a **Map** tab to the main view that links to a new route
- a map to start building on
- a collapsible side bar that has a placeholder title and content

## Testing
**URL to test:** 

https://deploy-preview-1276--atd-moped-main.netlify.app/moped/

**Steps to test:**
1. Go to the deploy preview and click the new navigation tab called **Map**. You should see that the sidebar is expanded by default.
2. Click the arrow in side menu bar to collapse and expand it. Scroll the placeholder area of the side bar and see that the title and nav arrow stay at the top.
3. Adjust the width to 600px or less or use a mobile device mode in dev tools. You should see that the sidebar is collapsed by default.
4. Click the nav arrow to expand it, scroll the placeholder area, and see that the title and arrow button stay at the top.

---
#### Ship list
